src/core/qgseditformconfig.cpp

```diff
@@ -119,6 +119,30 @@ bool QgsEditFormConfig::labelOnTop( int idx ) const
     return false;
 }
 
+QString QgsEditFormConfig::constraint( int idx ) const
+{
+  QString expr = "";
```

Comment:
No need to initialize a QString - it'll be empty by default. There's a few other uses of "" in this PR, they should be replaced by "QString()" as it's cheaper to construct.